### PR TITLE
Adds a new haiku in gundam.md

### PR DESCRIPTION
**Pull request by thanosades**

### CODE REVIEW

**Review by ltagliaferri — CHANGES_REQUESTED** [review]
Thank you for submitting this haiku! Please add breaks at the end of your lines — `<br>`. The last line also needs an extra syllable.

A haiku is composed of 3 lines. The first line is 5 syllables long, the second line is 7 syllables long, and the third line is 5 syllables long. Although the majority of the haikus here are written in English, you can submit a haiku in any human language.

Please consult our [contributing guidelines](https://github.com/do-community/cloud_haiku#contributing) if you need extra guidance. Once you amend this pull request, we can merge it in.

*Please note: after more than two days of inactivity, pull requests are subject to being closed and marked invalid by maintainers.*

**Comment by thanosades** [comment]
Omg I'm so sorry! I hope I did this right this time. I apologize for the inconvenience.

**Review by ltagliaferri — APPROVED** [review]
Awesome, thank you for sticking with this and updating, _and_ for being a part of open source 🎉

**Pull request merged**
